modulo5/LabECommerce/src/endpoints/RegisterPurchases.ts

```diff
@@ -0,0 +1,69 @@
+import { Request, Response } from "express";
+import connection from "../database/connection";
+import { Purchases } from "../types/types";
+
+
+
+export const RegisterPurchases = async (req: Request, res: Response): Promise<void> => {
```

Comment:
É bom manter sempre um padrão na declaração de funções e variaveis no projeto.  

```suggestion
export const registerPurchases = async (req: Request, res: Response): Promise<void> => {
```

Utilização de ferramentas como [eslint](https://eslint.org/) são usadas para manter um mesmo padrão de código dentro de todo o projeto.

Recomendo usar eslint junto com [prettier](https://prettier.io/) para facilitar a formatação dos arquivos de acordo com as regras definidas no eslint, assim você pode configurar o editor de texto para formatar os arquivos ao salvá-los, isso resolveria tbm o problema de identação e linhas sobrando.